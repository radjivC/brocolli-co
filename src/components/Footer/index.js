import React from 'react'

const Footer = () => {
  return (
    <footer className="footer">
      <div className="container">
        <div className="content has-text-centered">
          <p className="has-text-primary">
            <strong>Build in Melbourne</strong>
          </p>
        </div>
      </div>
    </footer>
  )
}

export default Footer
