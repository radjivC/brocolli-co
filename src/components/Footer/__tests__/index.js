/* global expectst */
import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../'

describe('Footer', () => {
  it('can render without crashing', () => {
    const component = renderer.create(
      <Footer />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
