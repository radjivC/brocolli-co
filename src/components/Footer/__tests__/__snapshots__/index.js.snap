// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer can render without crashing 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="content has-text-centered"
    >
      <p
        className="has-text-primary"
      >
        <strong>
          Build in Melbourne
        </strong>
      </p>
    </div>
  </div>
</footer>
`;
